core/src/test/resources/com/google/googlejavaformat/java/testimports/A.formatting-and-unused-import-removal

```diff
@@ -6,7 +6,6 @@ import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 import java.util.List;
-
```

Comment:
this seems to be an issue not covered before, as suppressed, assuming now the consistent format to be correct.

<img width="1554" alt="image" src="https://github.com/user-attachments/assets/d29e01cd-c59a-404f-8bbc-c3069a476337" />